config/appearance.php

```diff
@@ -18,6 +27,7 @@
 	'default-button-bg'            => $genesis_sample_link_color,
```

Comment:
Doesn't have to be for this PR, but for each `default-` here, they aren't always "default" once the colors are changed. Perhaps we should look at renaming these?